components/tokens/package.json

```diff
@@ -27,7 +27,7 @@
     "format:results": "prettier --no-config --no-error-on-unmatched-pattern --ignore-unknown --loglevel silent --write dist/"
   },
   "devDependencies": {
-    "@adobe/spectrum-tokens": "12.22.1",
+    "@adobe/spectrum-tokens": "0.0.0-s2-20230810215532",
```

Comment:
Ohh is this so we can use the S2 tokens?

Reply:
yes, at least for now, I took this from the [PR Patrick put up to test the s2 changes](https://github.com/adobe/spectrum-css/pull/2129)

Reply:
Yep! The one in the diff above is from one of the first snapshots that was released. Garth has also been releasing newer versions against the `13.0.0` number here: https://github.com/adobe/spectrum-tokens/releases - anything that's a `beta` for `13.0.0` is S2-related.